.github/workflows/ci.yml

```diff
@@ -120,4 +120,20 @@ jobs:
             make || exit 1; \
             cd ..; \
           done;
-
+  cmake-windows-latest:
+    runs-on: windows-latest
+    env:
+      APACHE_LOUNGE_DISTRO_VERSION: 2.4.57
+      WORKSPACE: 'D:\a\mod_proxy_cluster\mod_proxy_cluster\'
+      HTTPD_DEV_HOME: 'D:\a\mod_proxy_cluster\mod_proxy_cluster\httpd-apache-lounge\Apache24'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Let's add a human readable name for this step, e.g. "Setup Developer Command Prompt for Microsoft Visual C++"

Reply:
Done, thank you.